catalog/spatial/registry/registry-federation-admin-impl/src/main/java/org/codice/ddf/registry/federationadmin/impl/AdminHelper.java

```diff
@@ -100,4 +111,50 @@ public String getName(Configuration config) {
         return configAdmin.getObjectClassDefinition(config)
                 .getName();
     }
+
+    public Map<String, Object> getFilterProperties() throws IOException {
+        Map<String, Object> props = new HashMap<>();
+        Configuration config = configurationAdmin.getConfiguration(REGISTRY_POLICY_PID, null);
+        Dictionary<String, Object> configProps = config.getProperties();
+        Map<String, Object> configMap = new HashMap<>();
+        if (configProps == null) {
+            configProps = new Hashtable<>();
+        }
+        Enumeration<String> keys = configProps.keys();
+        while (keys.hasMoreElements()) {
+            String key = keys.nextElement();
+            configMap.put(key, configProps.get(key));
+        }
+        props.put(FederationAdmin.CLIENT_MODE, configMap.getOrDefault(DISABLE_REGISTRY_KEY, false));
+        props.put(FederationAdmin.FILTER_INVERTED, configMap.getOrDefault(WHITE_LIST_KEY, false));
+        props.put(FederationAdmin.SUMMARY_FILTERED,
+                configMap.getOrDefault(REGISTRY_ID_KEY, new String[0]));
+        return props;
+    }
+
+    public void setFilteredNodeList(List<String> filterList)  {
+        setFilterProperty(REGISTRY_ID_KEY, filterList.toArray(new String[0]));
+    }
+
+    public void setFilterInverted(boolean inverted)  {
+        setFilterProperty(WHITE_LIST_KEY, inverted);
+    }
+
+    public void setClientMode(boolean clientMode)  {
```

Comment:
❓ Do these new methods need to be public? Would it be detrimental to make them protected?